_posts/2017/2017-11-06-reunification_first_release_announcement.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Reunification first release announcement"
+author: dbarrosop
+description: "New napalm with all the unified drivers is now officially here. Come and check it out."
```

Comment:
`all the unified drivers` that might be confusing a bit. Maybe something like `core drivers` or something else (though we then need to explain what are the core drivers etc etc).